docs/guides/observability/health.adoc

```diff
@@ -97,6 +97,8 @@ The Containerfile `+HEALTHCHECK+` instruction defines a command that will be per
 
 The above code depends on the values of the {project_name} options such as `http-relative-path` (`http-management-relative-path`) and `http-management-port`. In case those are changed the code needs to be modified accordingly.
 
+NOTE: the `+HEALTHCHECK+` shown is based upon the ready endpoint. This is appropriate for informational purposes or to determine if the container should receive requests. It should not be used to determine if the container needs restarted.
```

Comment:
Do we have information on how people use HEALTHCHECK? I tried the official Docker docs, but they state that a container is eventually considered "unhealthy", but didn't describe any actions.